txnkv/transaction/commit.go

```diff
@@ -247,6 +247,10 @@ func (action actionCommit) handleSingleBatch(c *twoPhaseCommitter, bo *retry.Bac
 	return nil
 }
 
+func (actionCommit) isInterruptable() bool {
+	return true
```

Comment:
The background commit requests after the tranasction state is decided could be regarded as `Uninterruptable` to avoid leaving locks.